src/Migration/Command/GenerateCommand.php

```diff
@@ -199,11 +200,13 @@ private function getGeneratorSettings(InputInterface $input, string $environment
         $migrationsPaths = is_array($migrationsPaths) ? $migrationsPaths : (array)$migrationsPaths;
 
         // No paths? That's a problem.
-        if (empty($migrationsPaths[0])) {
+        if (count($migrationsPaths) === 0) {
```

Comment:
Now this line could cause unwanted warnings. Please revert it.

Reply:
Adjusted, not 100% reverted as index `0` does not exist on the array when using namespaces.